external/include/mbedtls/config.h

```diff
@@ -2882,7 +2882,7 @@
 #undef MBEDTLS_ERROR_STRERROR_DUMMY
 #undef MBEDTLS_GENPRIME
 
-#undef MBEDTLS_FS_IO
```

Comment:
This might be conflict with our file system. It should be verified before disabled.